cycledash/static/js/examine/RecordStore.js

```diff
@@ -379,7 +379,7 @@ function createRecordStore(run, igvHttpfsUrl, dispatcher, opt_testDataSource) {
     var search = window.location.search.substring(1),
         vars = search.split('&');
     var val = _.first(_.filter(vars, v => {
-      var [key, val] = v.split('=');
+      var [key] = v.split('=');
```

Comment:
This doesn't seem like a clear change (nor the one above with `[subquery]`; there should be some indication that we are indeed unpacking two values (like we'd use a `_` in Python), even if we're only using one of them.